modules/admin_manual/pages/configuration/server/occ_command/file_commands.adoc

```diff
@@ -230,30 +236,109 @@ TIP: If many users are affected, it could be convenient to create a shell script
+This is useful before removing a user.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} files:transfer-ownership --help
+ Usage:
+   files:transfer-ownership [options] [--] <source-user> <destination-user>
+----
+
+=== Arguments
+
+[width="100%",cols="20%,70%",]
+|===
+| `source-user`      | owner of files which shall be moved
+| `destination-user` | user who will be the new owner of the files
+|===
+
+=== Options
+
+[width="100%",cols="35%,70%",]
+|===
+| `--path=[PATH]`        | selectively provide the path to transfer. +
+For example --path="folder_name"
+| `-s, +
+--accept-skipped-shares` | always confirm to continue in case of skipped shares.
+|===
+
 For example, to move all files from `<source-user>` to `<destination-user>`, use the following command:
 
 [source,console,subs="attributes+"]
 ----
-{occ-command-example-prefix} files:transfer-ownership <source-user> <destination-user>
+{occ-command-example-prefix} files:transfer-ownership \
+    <source-user> \
+    <destination-user>
 ----
 
-You can also move a limited set of files from `<source-user>` to `<destination-user>` by making use of the `--path` switch, as in the example below. 
+You can also move a limited set of files from `<source-user>` to `<destination-user>` by making use of the `--path` switch, as in the example below.
 Ownership of `folder/to/move` and all files and folders which it contains will be transferred to `<destination-user>`.
 
 [source,console,subs="attributes+"]
 ----
-{occ-command-example-prefix} files:transfer-ownership --path="folder/to/move" \
+{occ-command-example-prefix} files:transfer-ownership \
+    --path="folder/to/move" \
     <source-user> \
     <destination-user>
 ----
 
-When using this command, please keep the following in mind:
+Please keep the following in mind when using this command:
 
 . The directory provided to the `--path` switch *must* exist inside `data/<source-user>/files`.
-. The directory and its contents won't be moved as-is between the users. 
-  It will be moved into the destination user's `files` directory, into a directory name which follows the format: `transferred from <source-user> on <timestamp>`. 
+. The directory and its contents won't be moved as-is between the users.
+  It will be moved into the destination user's `files` directory, into a directory name which follows the format: `transferred from <source-user> on <timestamp>`.
   Using the example above, it will be stored under: `data/<destination-user>/files/transferred from <source-user> on 20170426_124510/`
-. Currently file versions can't be transferred. 
+. Currently file versions can't be transferred.
   Only the latest version of moved files will appear in the destination user's account.
+
+== The files:troubleshoot-transfer-ownership command
+
+This command is used to scan for problems, that might have occurred during a run of
+ownership transfer using the above command set `files:transfer-ownership`.
```

Comment:
```suggestion
ownership transfer using the above command `files:transfer-ownership`.
```

I don't understand why the word "set" is there.

Reply:
Thanks for catching it